### PR TITLE
feat: Restructure README

### PR DESCRIPTION
**Pull request by dushyantbehl**
### Description of the change

Follwing the internal discussion we decided to restructure README into separate sections to make it smaller. This PR converts the README into three different pages making the main README smaller and more readable.

### Was the PR tested
NA

### CODE REVIEW

File: docs/installations.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Table of Contents
+
+Table of contents:
+ - [Basic Installations](#basic-installation)
+ - [Installing FlashAttention](#using-flashattention)
+ - [Installing Fms Acceleration](#using-fms-acceleration)
+ - [Installing Mamba Model Support](#training-mamba-models)
```
[the remark this answers is not in the document]
Comment (dushyantbehl) [reply docs/installations.md:7]: fixed

File: docs/installations.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+```
+
+## Using Experiment Trackers
+
+To use experiment tracking with popular tools like [Aim](https://github.com/aimhubio/aim), note that some trackers are considered optional dependencies and can be installed with the following command:
+```
+pip install fms-hf-tuning[aim]
+```
+For more details on how to enable and use the trackers, Please see, [the experiment tracking section below](#experiment-tracking).
+
```
[the remark this answers is not in the document]
Comment (dushyantbehl) [reply docs/installations.md:48]: fixed

File: README.md [hunk 3]
```diff
@@ -32,44 +21,91 @@ This repo provides basic tuning scripts with support for specific models. The re
...
+Llava 1.6-34B  | LlavaNextForConditionalGeneration | ✔️ | ✔️ |
+Llava 1.5-7B  | LlavaForConditionalGeneration | ✅ | ✅ |
+Llava 1.5-13B  | LlavaForConditionalGeneration | ✔️ | ✔️ |
+
+**Note**:
+* vLLM currently does not support inference with LoRA-tuned vision models. To use a tuned LoRA adapter of vision model, please merge it with the base model before running vLLM inference.
+
+## Training and Training Parameters:
+
+Please refer our [document](./docs/training.md) to see how to start [Single GPU](./docs/training.md#single-gpu) or [Multi GPU]((./docs/training.md#multi-gpu) runs with fms-hf-tuning.
```
[the remark this answers is not in the document]
Comment (dushyantbehl) [reply README.md:96]: fixed

File: README.md [hunk 4]
```diff
@@ -32,44 +21,91 @@ This repo provides basic tuning scripts with support for specific models. The re
...
+
+You can also refer the same [document](./docs/training.md#tips-on-parameters-to-set) on how to use various training arguments.
+
+## Tuning Techniques:
+
+Please refer to our [Tuning document](./docs/tuning-techniques.md) for details on how to perform -
```
[the remark this answers is not in the document]
Comment (dushyantbehl) [reply README.md:102]: fixed

File: README.md [hunk 5]
```diff
@@ -32,44 +21,91 @@ This repo provides basic tuning scripts with support for specific models. The re
...
+## Training and Training Parameters:
+
+Please refer our [document](./docs/training.md) to see how to start [Single GPU](./docs/training.md#single-gpu) or [Multi GPU]((./docs/training.md#multi-gpu) runs with fms-hf-tuning.
+
+You can also refer the same [document](./docs/training.md#tips-on-parameters-to-set) on how to use various training arguments.
+
+## Tuning Techniques:
+
+Please refer to our [Tuning document](./docs/tuning-techniques.md) for details on how to perform - 
+* [LoRA](./docs/tuning.md#lora-tuning-example)
```
[the remark this answers is not in the document]
Comment (dushyantbehl) [reply README.md:103]: fixed

File: README.md [hunk 6]
```diff
@@ -32,44 +21,91 @@ This repo provides basic tuning scripts with support for specific models. The re
...
+* [Activated LoRA](./docs/tuning.md#activated-lora-tuning-example)
+* [GPTQ-LoRA](./docs/tuning.md#gptq-lora-with-autogptq-tuning-example) 
+* [Full Fine Tuning](./docs/tuning.md#fine-tuning)
+* [Use FMS Acceleration](./docs/tuning.md#fms-acceleration)
```
[the remark this answers is not in the document]
Comment (dushyantbehl) [reply README.md:107]: fixed

File: README.md [hunk 7]
```diff
@@ -1140,28 +373,9 @@ python main.py \
...
-Experiment tracking in fms-hf-tuning allows users to track their experiments with known trackers like [Aimstack](https://aimstack.io/), [MLflow Tracking](https://mlflow.org/docs/latest/tracking.html) or custom trackers built into the code like
-[FileLoggingTracker](./tuning/trackers/filelogging_tracker.py)
-
-The code supports currently two trackers out of the box, 
-* `FileLoggingTracker` : A built in tracker which supports logging training loss to a file.
-* `Aimstack` : A popular opensource tracker which can be used to track any metrics or metadata from the experiments.
-* `MLflow Tracking` : Another popular opensource tracker which stores metrics, metadata or even artifacts from experiments.
-
-Further details on enabling and using the trackers mentioned above can be found [here](docs/experiment-tracking.md).  
-
```
[the remark this answers is not in the document]
Comment (praveenj83) [reply README.md:1163]: For e.g., mlflow tracking / aim, is there an optional parameter that needs to be passed during installation? I dont see an installation command for experiment tracking support

File: README.md [hunk 8]
```diff
@@ -1,305 +1,67 @@
 # FMS HF Tuning
 
 - [Installation](#installation)
```
Comment (praveenj83) [inline README.md:3]: Broken link. Please change 'installations' to 'installation' everywhere.

**Review by praveenj83 — COMMENTED** [review]
Some fixes were missed
